<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>768</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>OpenSkyStacker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>4</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #333;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <widget class="QTableView" name="imageListView">
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="wordWrap">
       <bool>false</bool>
      </property>
      <property name="cornerButtonEnabled">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="StackerGraphicsView" name="imageHolder">
      <property name="styleSheet">
       <string notr="true">background-color: #222;</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1" rowspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    color: white;
}

QPushButton {
    color: white;
    background-color: #222;
    border-radius: 5px;
    padding: 4px;
}

QPushButton:hover {
    background-color: #2d74da;
}

QPushButton:disabled {
    color: #444;
    background-color: #777;
}</string>
        </property>
        <property name="title">
         <string>Image Selection</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <widget class="QPushButton" name="buttonSelectLightFrames">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Add Light Frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSelectDarkFrames">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add Dark Frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSelectDarkFlatFrames">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add Dark Flat Frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSelectFlatFrames">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add Flat Frames</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonSelectBiasFrames">
           <property name="enabled">
            <bool>true</bool>
           </property>
           <property name="text">
            <string>Add Bias Frames</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="styleSheet">
         <string notr="true">QWidget {
    color: white;
}

QPushButton {
    color: white;
    background-color: #222;
    border-radius: 5px;
    padding: 4px;
}

QPushButton:hover {
    background-color: #2d74da;
}

QPushButton:disabled {
    color: #444;
    background-color: #777;
}</string>
        </property>
        <property name="title">
         <string>Stacking</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QPushButton" name="buttonOptions">
           <property name="text">
            <string>Options</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="buttonStack">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Align and Stack</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>22</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: #333;</string>
   </property>
   <property name="nativeMenuBar">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="styleSheet">
    <string notr="true">background-color:#444;
border-top: 2px solid #555;</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StackerGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>ui/stackergraphicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
